### DISCUSSION TITLE
Livewire V3

### DISCUSSION THREAD

**Discussion by kmaphane** [MohmmedAshraf/laravel-translations#41] [General]
Good day, 

Just curious, does this package work with Livewire V3. 

Tried to install it and failed for the following reasons:

   Problem 1
    - Root composer.json requires outhebox/laravel-translations 0.0.7 -> satisfiable by outhebox/laravel-translations[0.0.7].
    - outhebox/laravel-translations 0.0.7 requires livewire/livewire ^2.10 -> found livewire/livewire[v2.10.0, ..., v2.12.6] but it conflicts with your root composer.json require (^v3.0.0).

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.

Your feedback will be appreciated. I tried with all dependencies as well. Thanks

**Comment by MohmmedAshraf** [comment]
Hey @kmaphane,

I've got a plan to support Livewire v3 as well. Stay tuned for the next releases!

Thanks

**Reply by sbc640964** [reply]
What is date expected for livewire 3 support?

**Reply by MohmmedAshraf** [reply]
I'll try to save time to release it next week

**Comment by rivalex** [comment]
Hi @MohmmedAshraf, any news about Livewire 3 support?

**Comment by rivalex** [comment]
Hi @MohmmedAshraf, any plans for Livewire 3 support?

**Comment by MohmmedAshraf** [comment]
@rivalex Hi i'm trying to save time to make it happen i appreciate your patience.

**Comment by MohmmedAshraf** [comment]
@kmaphane @rivalex released

**Reply by rivalex** [reply]
Excellent!!!!! tomorrow super testing! Thanks
